pages/[...slug].tsx

```diff
@@ -152,6 +152,7 @@ export default function Page({ node, nav, footer }: PageProps) {
         <meta property="og:url" content={metaUrl} />
         <meta property="og:image" content={metaImage} />
       </Head>
+      <a id="content" tabIndex={-1}></a>
```

Comment:
This seems to be an empty link instead of using the "skip-to-main-content" variable. Is this according to the accessibility guidelines?